content/user/admin_pages/tools/developers_tool_kit.md

```diff
@@ -8,10 +8,10 @@ weight: 90
 The Developers Tool Kit is an essential tool for anyone involved in customizing Zen Cart.
 
 
-The Developers Tool Kit will search through all the files and directories under the directory where you installed Zen Cart for an exact result of the entered text.
+The Developers Tool Kit will search through all the files and directories under the directory where you installed Zen Cart for an exact match with the entered text.
 
 
-You probably already have a tool that searches through files installed on your operating system, but the Developers Tool Kit has some features specifically related to Zen Cart.
+Your operating system probably already has a tool that searches through files, but the Developers Tool Kit has some features specifically related to Zen Cart.
```

Comment:
Not a fan of putting this on the OS. Sounds too geeky.

Reply:
Please tweak with your preferred wording. It was what was there before, I just reordered it to make it easier to read.